lib/update.js

```diff
@@ -0,0 +1,27 @@
+module.exports = update
+
+var Promise = require('lie')
+
+var updateOne = require('./helpers/update-one')
+var updateMany = require('./helpers/update-many')
+
+/**
+ * updates existing object.
+ *
+ * @param  {String}          prefix       optional id prefix
+ * @param  {String|Object}   idOrObject   id or object with `._id` property
```

Comment:
I think we should cover that `idOrObject`/`obectsOrIds` also accept Arrays and unify naming, shouldn't we?